### DISCUSSION TITLE
Option to send notification when TV series becomes partly avilable

### DISCUSSION THREAD

**Discussion by adamus1red** [sct/overseerr#2417] [Feature Requests]
When people request an upcoming season it seems it would make more sense to send the notification when it starts to become available rather than when the final episode airs.

**Comment by tgrosst** [comment]
Agreed. Unless it's a series that drops a whole season at once, you have to manually "Mark as Available" any future series that have been requested once the first episode airs. Otherwise, the user won't be notified until (potentially) months in the future when the last episode of that season airs.

**Comment by JKelloggs** [comment]
Would love this.
